### ISSUE TITLE
Add a hook to ts-fold

### ISSUE THREAD

**Issue by mattiasdrp** [jcs-elpa/ts-fold#15]
(I may write a lot of issues in the following days but don't mind them, I plan to push some PR to fix them ;-))

There is currently no hook attached to ts-fold so the way I load the modes looks like this:

```elisp

(use-package ts-fold
  :load-path "custom/ts-fold/"
  :hook
  (tuareg-mode . ts-fold-mode)
  (c++-mode    . ts-fold-mode)
  (python-mode . ts-fold-mode)
  )

(use-package ts-fold-indicators
  :load-path "custom/ts-fold/"
  :config
  (setq ts-fold-indicators-fringe 'left-fringe)
  (setq ts-fold-indicators-priority 100)
  :hook
  (c++-mode . ts-fold-indicators-mode)
  (tree-sitter-after-on-hook . ts-fold-indicators-mode)
  )
``` 

when I'd like to just write `(fold-mode . ts-fold-indicators-mode)`

Because otherwise the indicators don't appear unless I start editing my file since `ts-fold-indicators-mode` is hooked to `tree-sitter-after-on-hook`

**Comment by jcs090218** [comment]
> (I may write a lot of issues in the following days but don't mind them, I plan to push some PR to fix them ;-))

Great! I assumed those issues are #16, #17, #18?

> when I'd like to just write (fold-mode . ts-fold-indicators-mode)
>
> Because otherwise the indicators don't appear unless I start editing my file since ts-fold-indicators-mode is hooked to tree-sitter-after-on-hook

Hmmm... not quite sure the issue here. So `indicators` do appear after adding `ts-fold-indicators-mode` to `tree-sitter-after-on-hook`?

Would #19 fix the issue for you?

**Comment by mattiasdrp** [comment]
> Great! I assumed those issues are #16, #17, #18?

Exactly :-)

**Comment by jcs090218** [comment]
Merged #19. Close this now!

Thanks for the confirmation! :)
